### PR TITLE
Check out branch

### PR DESCRIPTION
**Pull request by abdelilahAchir**

### CODE REVIEW

File: mrscake/src/components/orderDetails/CartItem.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React, {useState} from 'react';
+import {connect} from 'react-redux';
+//import {shortenTitle} from "../../pipes/shortenTitle";
+//import {formatMoney} from "../../pipes/priceFormatter";
+import './CartItem.scss';
+import { decrementCartQuantity, incrementCartQuantity, removeProductToCart} from "../../actions";
+
```
Comment (Alina-dev-front) [inline mrscake/src/components/orderDetails/CartItem.js:5]: which component is it? Class or functional component?

File: mrscake/src/components/orderDetails/CartItem.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.quantity .minus {
+  bottom: 0
+}
+.shopping-cart {
+  margin-top: 20px;
+}
+
+@media only screen and (max-width: 768px) {
```
Comment (Alina-dev-front) [inline mrscake/src/components/orderDetails/CartItem.scss:69]: here is a code for mobile devices, but we dont adopt our web site for them. MAy be delete these classes?

File: mrscake/src/components/orderDetails/CartItem.js (hunk 1 above)
Comment (abdelilahAchir) [reply mrscake/src/components/orderDetails/CartItem.js:5]: its constant, has the same fucntion as a function

File: mrscake/src/components/orderDetails/CartItem.scss (hunk 2 above)
Comment (abdelilahAchir) [reply mrscake/src/components/orderDetails/CartItem.scss:69]: you can take it away if doesn't effect anything

File: mrscake/src/components/orderDetails/CartItem.js (hunk 1 above)
Comment (abdelilahAchir) [reply mrscake/src/components/orderDetails/CartItem.js:5]: it has the same function as function component

**Review by Alina-dev-front — APPROVED** [review]

**Pull request merged**